<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>951</width>
    <height>606</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>571</width>
      <height>551</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="acceptDrops">
        <bool>false</bool>
       </property>
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>                                                                                   КАРТИНКА</string>
       </property>
       <property name="textFormat">
        <enum>Qt::PlainText</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="acceptDrops">
        <bool>false</bool>
       </property>
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>                                                                                   КАРТИНКА</string>
       </property>
       <property name="textFormat">
        <enum>Qt::PlainText</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>200</y>
      <width>321</width>
      <height>321</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Отправить данные</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>60</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Загрузить картинку</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>951</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Режимы</string>
    </property>
    <widget class="QMenu" name="menu_3">
     <property name="title">
      <string>Видео режим</string>
     </property>
     <addaction name="action_7"/>
     <addaction name="action_8"/>
    </widget>
    <addaction name="menu_3"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Войти</string>
    </property>
    <addaction name="action_4"/>
    <addaction name="action_5"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_2">
   <property name="text">
    <string>Режим 2</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>Режим 3</string>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>Создать новый конфиг</string>
   </property>
  </action>
  <action name="action_4">
   <property name="text">
    <string>Выбрать файл</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_5">
   <property name="text">
    <string>Ввести данные</string>
   </property>
  </action>
  <action name="action_6">
   <property name="text">
    <string>Выбрать видео</string>
   </property>
  </action>
  <action name="action_7">
   <property name="text">
    <string>Распознать с видео файла</string>
   </property>
  </action>
  <action name="action_8">
   <property name="text">
    <string>Распознания с видео камеры</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>mybutton_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>370</x>
     <y>441</y>
    </hint>
    <hint type="destinationlabel">
     <x>229</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>sent_data()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>850</x>
     <y>311</y>
    </hint>
    <hint type="destinationlabel">
     <x>475</x>
     <y>302</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_4</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>Open_conf()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>475</x>
     <y>302</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_5</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>open_login()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>475</x>
     <y>302</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_8</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>video_real()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>475</x>
     <y>302</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_7</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>open_video()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>475</x>
     <y>302</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>mybutton_clicked()</slot>
  <slot>sent_data()</slot>
  <slot>Open_conf()</slot>
  <slot>open_login()</slot>
  <slot>video_real()</slot>
  <slot>open_video()</slot>
 </slots>
</ui>
